mobile/src/kernel/features.ts

```diff
@@ -16,6 +17,40 @@ export const features = {
   utxoConsolidation: isDev,
 }
 
+export const useYoroiConfig = () => {
+  const {data, isLoading, error} = useQuery({
+    queryKey: ['yoroi-config', isDev],
```

Comment:
this will store it locally
```suggestion
    queryKey: ['persist', 'yoroi-config', isDev],
```